### DISCUSSION TITLE
Should we update Wikipedia?

### DISCUSSION THREAD

**Discussion by goatandsheep** [atom-ide-community/atom#495] [Ideas]
Wikipedia says that Atom has been completely deprecated, but Brackets Wiki after Adobe left it is very healthy. Should we indicate that Atom is being maintained by the community here?

**Comment by AlexWayfer** [comment]
Atom Community is like paused or abandoned too. [Pulsar](https://pulsar-edit.dev/) is more active promising fork, but there are still a lot of work to do. Not sure what and where we should indicate.
